src/xcode/ENA/ENA/Source/Scenes/Checkin/QRCodeVerificationHelper.swift

```diff
@@ -83,6 +91,17 @@ class QRCodeVerificationHelper {
 			onError(CheckinQRScannerError.invalidCryptoSeed)
 			return
 		}
+		
+		let startTimeStamp = traceLocation.startDate?.timeIntervalSince1970 ?? 0
+		let endTimeStamp = traceLocation.endDate?.timeIntervalSince1970 ?? 0
+		let bothTimeStampsAreZero = (startTimeStamp == 0 && endTimeStamp == 0)
+		let startTimeIsBeforeEndTime = startTimeStamp < endTimeStamp
```

Comment:
In our current implementation it is possible to set startDate and endDate to be equal. 
```suggestion
		let startTimeIsBeforeEndTime = startTimeStamp <= endTimeStamp
```

Reply:
it says here it must be less, should i ignore it ?
<img width="830" alt="Screenshot 2021-04-13 at 15 23 09" src="https://user-images.githubusercontent.com/15270737/114559575-2a98f380-9c6c-11eb-8792-a09b456f28c0.png">
https://github.com/corona-warn-app/cwa-app-tech-spec/blob/f5ba0fdcaffc05e44df80067b8ec21d7ba721503/docs/spec/event-registration-client.md#attendee-check-in

Reply:
I think it's best to ask Max about it, I am not aware of a restriction like that for the trace location generation. We would need to change the implementation there as well if this was the new requirement.